engine/classes/Elgg/Exceptions/Plugin/IdMismatchException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Elgg\Exceptions\Plugin;
+
+use Elgg\Exceptions\PluginException;
+
+/**
+ * Indicates if the plugin id mismatches
```

Comment:
Indicates a mismatch between the plugin ID in the ``composer.json`` and the plugin directory